pg_variables.c

```diff
@@ -909,7 +913,7 @@ remove_variable(PG_FUNCTION_ARGS)
 	else
 		removeObject(&variable->transObject, TRANS_VARIABLE);
 
-	resetVariablesCache(false);
+	resetVariablesCache(true);
```

Comment:
It seems that resetVariablesCache() is used with `true` everywhere now. Are there any arguments to keep `with_package` flag in the resetVariablesCache() now?

Reply:
Only for historical reasons. I think we can get rid of this argument.